public/editor/scripts/editor/js/fc/bramble-ui-bridge.js

```diff
@@ -11,10 +11,72 @@ define(function(require) {
 
   var _escKeyHandler;
 
+  var adapting = false;
+  var adaptTimeout = 200; // How often we adapt editor bar layout
```

Comment:
Name variables like this with their associated unit, `adaptTimeoutMS` to indicate MS values.